recipes/glymur/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.8.3" %}
+package:
+  name: glymur
+  version: {{ version }}
+
+source:
+  fn: Glymur-0.8.3.tar.gz
+  url: https://pypi.python.org/packages/33/6e/3bb303c46e2b9a795389115401fd32f1eff16c12a66c7a53fa36c244f9bc/Glymur-0.8.3.tar.gz
+  md5: 2ff4c70a63a1e01f84b18e0ec4eb9b88
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py35]
+  entry_points:
+    - jp2dump=glymur.command_line:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.7.0
```

Comment:
i suppose the only use would be if someone has some custom packages coming from somewhere else. It's probably worth keeping.